README.md

```diff
@@ -705,13 +705,15 @@ OR
 (0 rows)
 ```
 
-## TODO
+## NOTES
 
-The following changes/enhancements are contemplated:
+### Version 2.0.0
 
-* Improve regression tests
-* Add ability to create dependencies in `shared_preload_libraries` such that
-  extension order does not matter.
+- Use of GUCs with `whitelist` have been deprecated in lieu of a more
+  specific `allowlist`. The old GUCs will still be used, but produce a warning
```

Comment:
I would say something like "in lieu of a more appropriately named `allowlist`" instead. The list itself is no more specific than before, we are just changing the name and that is not clear to me from the current wording.